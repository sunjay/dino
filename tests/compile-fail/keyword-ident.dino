fn main() {
    let let = 10;
}
