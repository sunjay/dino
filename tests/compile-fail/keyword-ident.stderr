[0m[1m[31merror: [0m[tests/compile-fail/keyword-ident.dino:1] expected an identifer, found: `let`

